### DISCUSSION TITLE
How to update Open-CRAVAT from v2.2.9 to v2.5.0 ?

### DISCUSSION THREAD

**Discussion by lmanchon** [KarchinLab/open-cravat#237] [Usability & Feature Requests]
--Hi,

is there some command lines to update old version v2.2.9 to 2.5.0 ?

thank you --

**Comment by kmoad** [comment]
How to upgrade the core OpenCRAVAT software depends on how you installed. For most users installing through pip, you upgrade through pip.

```
pip install --upgrade open-cravat
```

Then you should upgrade the oc modules

```
oc module update
```

Instructions for other platforms are in our docs https://open-cravat.readthedocs.io/en/latest/Update-Instructions.html
